civihr_default_theme/scss/pages/_onboarding-wizard.scss

```diff
@@ -1,4 +1,14 @@
 .onboarding-wizard-page {
+  #edit-submitted-add-another-emergency-contact {
+    .form-type-radio {
+      width: 50%;
+    }
+  }
+
+  label[for="edit-submitted-civicrm-1-contact-1-cg16-custom-73"] {
```

Comment:
is this a reliable selector? It seems auto-generated. What if it changes?

Reply:
Fixed